server/routing-test.js

```diff
@@ -57,19 +50,22 @@ describe('server routing', () => {
     }
 
     it('GET / renders OK', () => {
+      useDummyData();
       return get('/')
         .expect(/React London/)
         .expect(/Conference/)
         .expect(200);
     });
 
     it('GET /conference redirects to /', () => {
+      useDummyData();
       return get('/conference')
         .expect(302)
         .expect('LOCATION', '/');
     });
 
     it('GET /community redirects', () => {
+      useDummyData();
```

Comment:
Yes but I don't like beforeEach as they're easy to lose track of. I would much rather be explicit.